parsel/utils.py

```diff
@@ -80,4 +81,12 @@ def extract_regex(regex, text, replace_entities=True):
     strings = flatten(strings)
     if not replace_entities:
         return strings
-    return [w3lib_replace_entities(s, keep=['lt', 'amp']) for s in strings]
\ No newline at end of file
+    return [w3lib_replace_entities(s, keep=['lt', 'amp']) for s in strings]
+
+def selector_data_shorten(data, width):
+    """Shortens the preview of extracted data by adding placeholder '...'
+    """
+    if six.PY34:
```

Comment:
We don't want to use six.PY34, because that isn't available in all six versions (we support [six>=1.5.2](https://github.com/scrapy/parsel/blob/master/setup.py#L34), and 1.5.2 [doesn't have it](https://github.com/JioCloud/python-six/blob/1.5.2/six.py#L31-L33))

It would be better to do in the import section:

```
try:
    from textwrap import shorten
except ImportError:
    shorten = None
```

And then check `shorten is None` in here to decide what to do.